app/models/reply.rb

```diff
@@ -43,7 +43,44 @@ class Reply < ActiveRecord::Base
         .where('locked_by_id IN (?) OR locked_at < ?',
             [user.id, nil], Time.zone.now - 5.minutes)
   }
-
+  
+  def content_without_quotes
+    # 1. Remove custom style from email.
+    #
+    result = content
+      .gsub(/<style.*<\/style>/im, "")
+      .gsub(/<head.*<\/head>/im, "")
+    
+    # 2. Detect common ways that lead in email quotes or signatures.
+    #    http://stackoverflow.com/a/2193937/2066546
+    #
+    result = result
+      .gsub(/<blockquote.*<\/blockquote>/im, "")
+      .gsub(/----.*/m, "")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.